src/plasmapy/analysis/swept_langmuir/helpers.py

```diff
@@ -235,3 +237,285 @@ def sort_sweep_arrays(
         current = current[::-1]
 
     return voltage, current
+
+
+def _is_voltage_regularly_spaced(
+    voltage_diff: np.ndarray,
+    mask_zero_diff: np.ndarray,
+) -> bool:
+    """
+    Determine if the voltage difference array ``voltage_diff`` is
+    regularly spaced; that is the differences are all equal or some
+    integer multiple of the smallest difference.
+    """
+    is_regular_grid = False
+    if np.count_nonzero(mask_zero_diff) > 0:
+        # is_regular_grid = False
+        pass
+    elif np.allclose(voltage_diff, voltage_diff[0]):
+        # grid is already regularly spaced
+        is_regular_grid = True
+    elif np.allclose(
+        np.rint(voltage_diff / np.min(voltage_diff))
+        - voltage_diff / np.min(voltage_diff),
+        0,
```

Comment:
```suggestion
    elif np.allclose(voltage_diff, voltage_diff[0]):
        # grid is already regularly spaced
        is_regular_grid = True
    elif np.allclose(
        np.rint(voltage_diff / np.min(voltage_diff))
        - voltage_diff / np.min(voltage_diff),
        0,
```

Should we include `atol` and/or `rtol` explicitly?